tools/StaticAnalysis/Exceptions/Az.Network/SignatureIssues.csv

```diff
@@ -446,3 +446,4 @@
 "Az.Network","Microsoft.Azure.Commands.Network.UpdateAzureRmP2SVpnGatewayCommand","Update-AzP2sVpnGateway","1","8700","Parameter set 'ByP2SVpnGatewayResourceIdNoVpnServerConfigurationUpdate', 'ByP2SVpnGatewayResourceIdByVpnServerConfigurationObject' of cmdlet 'Update-AzP2sVpnGateway' have the same mandatory parameters, and both of them are not default parameter set which may cause confusion.","Merge these parameter sets into one parameter set."
 "Az.Network","Microsoft.Azure.Commands.Network.Bastion.NewAzBastionCommand","New-AzBastion","1","8410","Parameter EnableKerberos of cmdlet New-AzBastion does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
 "Az.Network","Microsoft.Azure.Commands.Network.Bastion.SetAzBastionCommand","Set-AzBastion","1","8410","Parameter EnableKerberos of cmdlet Set-AzBastion does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"Az.Network","Microsoft.Azure.Commands.Network.NewAzureApplicationGatewayFirewallPolicySettingCommand","New-AzApplicationGatewayFirewallPolicySetting","1","8410","Parameter JSChallengeCookieExpirationInMins of cmdlet New-AzApplicationGatewayFirewallPolicySetting does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
```

Comment:
@Shawnli222, can you rename the parameter using a singular noun instead of suppressing the issue?

Reply:
We have to make it the same as the swagger PR so it had to be like this